From 218a904ccad845d28f9c7314a882fba4adf21619 Mon Sep 17 00:00:00 2001
From: toca <lyle_wjy@163.com>
Date: Wed, 17 Jul 2019 17:31:06 +0800
Subject: [PATCH] imx6ull-14x14-evk.dtb->imx6ull-14x14-edk.dtb

---
 include/configs/mx6ullevk.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/mx6ullevk.h b/include/configs/mx6ullevk.h
index 6753d27..363b400 100755
--- a/include/configs/mx6ullevk.h
+++ b/include/configs/mx6ullevk.h
@@ -243,7 +243,7 @@
 				"if test $board_name = EVK && test $board_rev = 9X9; then " \
 					"setenv fdt_file imx6ull-9x9-evk.dtb; fi; " \
 				"if test $board_name = EVK && test $board_rev = 14X14; then " \
-					"setenv fdt_file imx6ull-14x14-evk.dtb; fi; " \
+					"setenv fdt_file imx6ull-14x14-edk.dtb; fi; " \
 				"if test $fdt_file = undefined; then " \
 					"echo WARNING: Could not determine dtb to use; fi; " \
 			"fi;\0" \
-- 
2.7.4

